test/functional/test_runner.py

```diff
@@ -472,9 +472,8 @@ def was_successful(self):
 
 
 def check_script_prefixes():
-    """Check that no more than `EXPECTED_VIOLATION_COUNT` of the
+    """Check that at most a handful of the
        test scripts don't start with one of the allowed name prefixes."""
-    EXPECTED_VIOLATION_COUNT = 77
 
     # LEEWAY is provided as a transition measure, so that pull-requests
     # that introduce new tests that don't conform with the naming
```

Comment:
Might as well set LEEWAY to 0, as I don't see a need for this.

After all, we *want* travis to fail if some test is named inproperly. Otherwise, it would lead to unnecessary fixup pull requests.

Reply:
Hmm, you might be right -- no specific example comes to mind, but I'm still a bit hesistant about removing LEEWAY prior to 0.16 branching from master.

I've put a commit that drops LEEWAY in #12252 ; assuming you definitely want it and Travis is happy with it, can merge it into this PR; otherwise can just merge #12252 when 0.16 branches off?